RNVoipPushNotification.podspec

```diff
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
   s.requires_arc        = true
   s.platform            = :ios, "8.0"
   s.source_files        = "ios/RNVoipPushNotification/*.{h,m}"
-  s.dependency 'React/Core'
+  s.dependency 'React-Core'
```

Comment:
Seems like it's a breaking change, could you test if works for you if you change it

from:
`s.dependency 'React-Core'`
to
`s.dependency 'React'`
?

Reply:
I can confirm it works with both but it looks like other packages us `s.dependency 'React'` as I think it's backwards compatible